### DISCUSSION TITLE
What is the difference between winversion and linversion for Krokiet v8.0.0 Windows binaries?

### DISCUSSION THREAD

**Discussion by marcushallett** [qarmin/czkawka#1371] [Q&A]
In the version 8.0.0 release, there are multiple Krokiet executables for Windows:

1. `windows_krokiet_gui_winversion.exe`
2. `windows_krokiet_gui_linversion.exe`
3. `windows_krokiet_gui_linversion_console.exe`

Obviously the third shows a console window. What is the difference between the winversion and linversion binaries? I don't see any differences in the running applications and wasn't able to find any documentation about this from a quick search.

**Comment by qarmin** [comment]
This is quite new thing, so I added now to legend in release page this explanation

"""
winversion, linversion - the application has been compiled on Windows or cross-compiled on Linux. Both versions should work exactly the same, but the Windows version requires the [Microsoft Visual C++ Redistributable](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version) package to be installed (it 
"""
